src/main/java/com/xiaohengjin/Pojo2ProtoCore.java

```diff
@@ -70,6 +78,38 @@ public void go(PsiClass clazz) {
         lines.addAll(protoContentList);
     }
 
+    private void goEnum(PsiClass clazz) {
+        String className = clazz.getName();
+        List<String> protoContentList = new ArrayList<>();
+        protoContentList.add("enum " + convertClassName(className) + " {");
+        PsiField[] fields = clazz.getFields();
+        int index = 0;
+        // As per protobuf convention - enums should have a NONE field.
+        protoContentList.add(getFieldNameForEnum(className, "NONE", index++));
+        for (PsiField field: fields) {
+            // Enum values are copied as-is
+            protoContentList.add(getFieldNameForEnum(className, field.getName(), index++));
+        }
+
+        protoContentList.add("}\n\n");
+        System.out.println(StringUtils.join(protoContentList, "\n"));
+        lines.addAll(protoContentList);
+    }
+
+    @VisibleForTesting
+    String getFieldNameForEnum(String enumName, String fieldName, int index){
+        enumName = CaseFormat.UPPER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, enumName);
+        if(!fieldName.toUpperCase().equals(fieldName)) {
```

Comment:
code format: if and () should have a blank

Reply:
Done.